net/curl/Portfile

```diff
@@ -34,7 +34,7 @@ checksums                       ${curl_distfile} \
 if {${name} eq ${subport}} {
     PortGroup                   muniversal 1.0
 
-    revision                    0
+    revision                    1
```

Comment:
The revision should not be increase because this doesn't change the files that will get installed for anyone who already has it installed.

Reply:
Is there another way to trigger a rebuild, or do the bots just do that when you change the file?

Reply:
The buildbot will build and upload anything that has not already been built and uploaded. When I wrote that comment, you were proposing changing the default variant to +darwinssl. The set of variants is part of the archive file name so in this case the buildbot would check if an archive for this version and revision of curl with +darwinssl had already been uploaded, would find that it had not been, and would build and upload it.

Since then, you've changed the PR to propose that the +ssl variant be removed. +ssl was formerly a default variant so if you use this new proposal then yes the revision should be increased.

Reply:
In fact even if it had not been a default variant, removing a variant always requires increasing the revision, otherwise there could still be users out there with that variant installed.